.github/workflows/create-jira-issues-for-community-activities.yml

```diff
@@ -0,0 +1,43 @@
+name: Create Jira issues for community activities
+
+on:
+  workflow_call:
+    secrets:
+      # We can't access org secrets here so they need to be passed in.
+      JIRA_BASE_URL:
+        required: true
+      JIRA_USER_EMAIL:
+        required: true
+      JIRA_API_TOKEN:
+        required: true
+      JIRA_PROJECT_KEY:
+        required: true
+      DISCUSSION_JIRA_ISSUE_DESCRIPTION:
+        required: true
+      ISSUE_JIRA_ISSUE_DESCRIPTION:
+        required: true
+      PULL_REQUEST_JIRA_ISSUE_DESCRIPTION:
+        required: true
```

Comment:
Can't we provide sensible defaults for these? I understand that you don't want to include our issue completion checklist URL, but is that applicable for the DISCUSSION and PULL_REQUEST types too? Also we could provide a default without the offending link for issue description anyway, in case anyone else wants to use the workflow.

Reply:
This was a fun hour wasted, but apparently there's no way to override env vars GitHub sets. Anyway, I've did it otherwise.